src/ui/controls/edit.h

```diff
@@ -132,8 +124,13 @@ class CEdit : public CControl
     bool        ReadText(std::string filename, int addSize=0);
     bool        WriteText(std::string filename);
 
-    void        SetMaxChar(int max);
-    int         GetMaxChar();
+    void        SetMaxChar(int max);
+    //! Get max characters that can be hold before truncation or if is Infinite edit actual allocated characters in m_text
+    int         GetMaxChar();
+    //! Has no character limit for edit
+    bool        IsInfinite();
+    //! Set no character limit for edit
+    void        SetInfinite();

```

Comment:
I don't think having a separate function for this is really necessary. Infinite should be the default, and if you ever need to revert SetMaxChar setting, you can always just do `SetMaxChar(std::numeric_limits<int>::max())`